### ISSUE TITLE
Implement from Scratch & Replace PeerJS

### ISSUE THREAD

**Issue by aswinshenoy** [aswinshenoy/bayjdo#5]

**Comment by aswinshenoy** [comment]
The 'simple-peer' looks good, and I shall try and migrate to it if possible. However, in the near future, I am planning to keep PeerJS working and work on other critical issues on the app

**Comment by aswinshenoy** [comment]
`simple-peer` - `socket.io` combination could be used for transferring P2P & in small-groups, whereas webtorrent can be used for larger & persistent file sharing

**Comment by BRAVO68WEB** [comment]
Yeah socket.io will be much faster for but may (1% chance) give invalid requests.

But yes it will be very Good

**Comment by aswinshenoy** [comment]
I am currently busy with another project, so do not think I will be working on these anytime soon. I would be very happy if anyone could be help BayJDO out with these :)

**Comment by BRAVO68WEB** [comment]
If possible I may help out in future 😃😃
 I am busy too
